pkg/daemon/ceph/osd/daemon.go

```diff
@@ -50,13 +50,19 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID string, pvcBack
 	}
 
 	if pvcBackedOSD {
+		logger.Infof("LV Path for OSD %s : %+v", osdID, lvPath)
+		vgName := strings.Split(lvPath, "/")[2]
 		if err := updateLVMConfig(context); err != nil {
 			return fmt.Errorf("sed failure, %+v", err) // fail return here as validation provided by ceph-volume
 		}
 
-		context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-an")
+		if err := context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-an", vgName); err != nil {
```

Comment:
From readability perspective, it'd be easier to write '--activate' and not '-a'.

What about '--force' ?